definitions.go

```diff
@@ -20,14 +24,21 @@ func GetResourceDefinitionTasks(tasks chan<- Task, runner Runner, projects, reso
 // definition for the given resource in the given project.
 func ResourceDefinition(r Runner, project, resource string) Task {
```

Comment:
as project can be empty, does it make more sense to make it the last argument?
i.e. 
```
ResourceDefinition(runner, "persistentvolumes", "")
```
Rather than:
```
ResourceDefinition(runner, "", "persistentvolumes")
```

or possibly we could wrap the method with something like:
```
func PlatformResourceDefinition(r Runner, resource string) Task {
    return ResourceDefinition(r, resource, "")
}
```
What do you think?